cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataProcProvisioner.java

```diff
@@ -44,7 +45,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(DataProcProvisioner.class);
   private static final ProvisionerSpecification SPEC = new ProvisionerSpecification(
-    "gce-dataproc", "Google Cloud Dataproc Provisioner",
+    "gcp-dataproc", "Google Cloud Dataproc Provisioner",
```

Comment:
Can you also remove the word "Provisioner" from here?

Reply:
there's a separate jira for that